components/brave_ads/core/internal/account/utility/redeem_confirmation/reward/redeem_reward_confirmation.cc

```diff
@@ -105,7 +107,11 @@ void RedeemRewardConfirmation::CreateConfirmationCallback(
   BLOG(6, UrlResponseToString(url_response));
   BLOG(7, UrlResponseHeadersToString(url_response));
 
-  FetchPaymentToken(std::move(redeem_confirmation), confirmation);
+  // Delay fetching payment token to allow the server to create payment token.
+  PostDelayTaskToCurrentSequence(
+      base::BindOnce(&RedeemRewardConfirmation::FetchPaymentToken,
+                     std::move(redeem_confirmation), confirmation),
+      kFetchPaymentTokenAfter.Get());
```

Comment:
Move to a function such as `FetchPaymentTokenAfter`, removing the need for the comment